paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/BankFormScreenState.kt

```diff
@@ -41,8 +50,14 @@ internal data class BankFormScreenState(
         val financialConnectionsSessionId: String?,
         val mandateText: ResolvableString,
         val isVerifyingWithMicrodeposits: Boolean,
+        val eligibleForIncentive: Boolean = false,
```

Comment:
This is currently hard-coded to false, but will eventually be set in the bank auth flow.